test/jdk/java/lang/instrument/NativeMethodPrefixApp.java

```diff
@@ -34,27 +31,25 @@
 
 /*
  * @test
- * @bug 6263319
+ * @bug 6263319 8334167
  * @summary test setNativeMethodPrefix
  * @requires ((vm.opt.StartFlightRecording == null) | (vm.opt.StartFlightRecording == false)) & ((vm.opt.FlightRecorder == null) | (vm.opt.FlightRecorder == false))
- * @modules java.management
- *          java.instrument
+ * @modules java.instrument
  * @library /test/lib
  * @build bootreporter.StringIdCallback bootreporter.StringIdCallbackReporter
  *        asmlib.Instrumentor NativeMethodPrefixAgent
  * @enablePreview
  * @comment The test uses asmlib/Instrumentor.java which relies on ClassFile API PreviewFeature.
- * @run driver/timeout=240 NativeMethodPrefixApp roleDriver
- * @comment The test uses a higher timeout to prevent test timeouts noted in JDK-6528548
+ * @run main/native NativeMethodPrefixApp roleDriver
  */
 public class NativeMethodPrefixApp implements StringIdCallback {
 
     // This test is fragile like a golden file test.
     // It assumes that a specific non-native library method will call a specific
     // native method.  The below may need to be updated based on library changes.
-    static String goldenNativeMethodName = "getStartupTime";
-
+    static String goldenNativeMethodName = "fooBarNativeMethod";
```

Comment:
The comment block above seems no longer applicable now this is not a library class method.

Reply:
I've now updated the code comment to note what that constant is for.